docs/index.md

```diff
@@ -130,31 +132,30 @@ The URL will be (update the GitHub username variable):
    ```  
```

Comment:
Because we're appending the component and repository with '-nodejs' in backstage we would need to add this to the URL